components/automate-gateway/api/compliance/reporting/stats/stats.proto

```diff
@@ -11,7 +11,13 @@ import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 import "components/automate-gateway/api/compliance/reporting/reporting.proto";
 
 service StatsService {
-	// should cover /summary, /summary/nodes, /summary/controls
+	/*
+	Read Summary
+
+	Returns summary statistics for compliance reports. 
+	General report summary information is the default. Adding a type value of nodes or controls will return summary statistics for that object.
+	Supports filtering.
+	*/
```

Comment:
We haven't really established any conventions, but this supports markdown so you can do

> Adding a `type` value of `nodes` or `controls`

etc to show what are keys/values.